src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -76,12 +76,9 @@ protected function getSerializer(): SerializerInterface
     }
 
     /**
-     * An instance of the Session implementation (and not the interface) is returned because getFlashBag is not part of
-     * the interface.
-     *
      * @required
      */
-    protected function getSession(): Session
+    protected function getSession(): SessionInterface
```

Comment:
This change violates the Liskov substitution principle, `getFlashBag()` is not part of this interface.